docs: add champion definition (#5)
NodeSecure/rfcs: README.md

There are currently no predefined rules for new ideas (we will probably work on that eventually).

However, if we decide to implement an idea we need at least one to two champions that will lead the subject. This means that we will not launch a new idea without having a champion to carry out.

### Champion

> A champion is someone who takes responsibility for the realisation of an idea. A Champion will by default become the maintainer of the project or tool it creates.

Reviewer: ```suggestion
A champion is someone who leads the realisation of an idea or a project. A Champion will most likely have the opportunity to become the maintainer of his own project or tool created.
```

### Champion

> A champion is someone who leads the realisation of an idea or a project. A Champion will most likely have the opportunity to become the maintainer of his own project or tool created.

Reviewer 2: ```suggestion
A champion is someone who leads the realization of an idea or a project. A Champion will most likely have the opportunity to become the maintainer of its project or tool created.
```
